pages/dashboard.js

```diff
@@ -173,9 +173,10 @@ const Dashboard = () => {
       <Section
         backgroundColor='lightRock'
         display={{ md: 'grid' }}
-        gridTemplateColumns={{ lg: 'minmax(16rem, 20rem) 1fr' }}
+        gridTemplateColumns={{ lg: 'minmax(16rem, 20rem) minmax(auto, 70rem)' }}
```

Comment:
why is the first column 'minmax(16rem, 20rem)' ? isn't that pretty large for the data cards?

Reply:
It's the same size as you see in production now. All I'm changing here is setting a max width for the chart section.

Reply:
yea i get that, but i dont understand how min width for the cards is 16rem. i see that it somehow looks ok, i just don't get it

Reply:
16rem is 256px and the smallest mobile screen is ~ 320px, so 16rem is less than full width on mobile.

Reply:
oh wow i didn't realize that. i guess i was thinking 16rem was way bigger! nvm